package.json

```diff
@@ -8,6 +8,8 @@
   "scripts": {
     "prepare": "husky install",
     "---SCRIPTS---": "",
+    "const-mainnet": "node ./scripts/createConstants.js mainnet",
```

Comment:
Lets keep it in the same form as other commands: `generate-const:holesky`, `generate-const:mainnet`. Also, I think we can put them under `"---HOLESKY---"` and `"---MAINNET---"`. Also, add them as part of the `prepare:holesky` and `prepare:mainnet`